openslides_backend/presenter/get_mediafile_id.py

```diff
@@ -0,0 +1,60 @@
+from typing import Any, Dict
+
+from .base import BasePresenter
+from .presenter import register_presenter
+from ..models.mediafile import Mediafile
+from ..shared.filters import FilterOperator
+import fastjsonschema
+from ..shared.schema import schema_version
+from ..shared.exceptions import PresenterException
+
+
+get_mediafile_id_schema = fastjsonschema.compile(
+    {
+        "$schema": schema_version,
+        "type": "object",
```

Comment:
Please add short title and description here (maybe we want to use this schema later also for documentation purposes).